<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatashowClass</class>
 <widget class="QMainWindow" name="DatashowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datashow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>标题行：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="title_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>跳过行：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="jump_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>结束行：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="end_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>X轴列：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>无效值：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="invalid_value"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_confirm">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reset">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_keeprevise">
        <property name="text">
         <string>保存修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_begin">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="layout_show">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>选择显示的数据：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_reverse">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>选择反转的数据：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
